backend/app/assets/javascripts/spree/backend/gateway.js

```diff
@@ -0,0 +1,22 @@
+Spree.ready(function() {
+  var $gateway_type = $('select.js-gateway-type');
+  var $preference_source = $('select.js-preference-source');
+  var original_gtwy_type = $gateway_type.val();
```

Comment:
what about taking advantage of this PR to change this variable name to `original_gateway_type`?